### PR TITLE
Make Traceur work with V8 block bindings

### PR DESCRIPTION
**Pull request by arv**
V8 does not support let/const inside switch statements. It requires
that you add an extra block (which is sane anyway).

This was tested with adding --block-binding=parse to the Makefile.

### CODE REVIEW

**Comment by arv** [comment]
@johnjbarton PTAL

File: src/codegeneration/CommonJsModuleTransformer.js [hunk 1]
```diff
@@ -129,10 +129,11 @@ export class CommonJsModuleTransformer extends ModuleTransformer {
       let descriptor;
 
       switch (exp.type) {
-        case GET_ACCESSOR:
+        case GET_ACCESSOR: {
```
Comment (johnjbarton) [inline src/codegeneration/CommonJsModuleTransformer.js]: Why add braces here (but not elsewhere)?

Comment (johnjbarton) [inline src/codegeneration/CommonJsModuleTransformer.js]: Oh, I didn't realize the commit description was referring to Traceur source rather than output.

Comment (johnjbarton) [inline src/codegeneration/CommonJsModuleTransformer.js]: Nevertheless...

**Comment by johnjbarton** [comment]
LGTM.  We'll break this if we don't adopt the style.

**Comment by arv** [comment]
It is really a V8 bug but I expect linters to disallow block bindings here. We have historically been opposed to making traceur add lint checks.

**Pull request merged**